### ISSUE TITLE
www.de

### ISSUE THREAD

**Issue by Daxy7** [easylist/easylist#18616]
parked website, redirects to ads
```
http://www.de/
```
```
https://search.infosayz.com/searchresults/?q=tvs
```
```
||infosayz.com^
```
second redirect
```
http://ww38.www.de/
```
```
! leftover ad scripts/css
||d25hvf57b1t0vp.cloudfront.net^
! tracker after accepting cookies
/track.php?toggle=consent&uid=[data_expunged]&domain=www.de&answer=timeoutLoading&token=
```
https://github.com/easylist/easylist/assets/91396482/bd6fbcbd-49a1-4605-9102-5155722f7f2b

**Comment by Khrin** [comment]
Not seen.

**Comment by Daxy7** [comment]
Redirects are a coin flip, Try UK/DE IP maybe.
https://github.com/easylist/easylist/assets/91396482/7208951c-d99b-4047-95b2-eecdca029e19

**Comment by Daxy7** [comment]
Domains by `parklogic.com` always have this `/track.php` xhr request enabled, to be block globally
```
http://ww12.chatgptbeta.com/
```
```
http://ww12.chatgptbeta.com/track.php?toggle=consent&uid=&domain=chatgptbeta.com&answer=timeoutLoading&token=
```

**Comment by Daxy7** [comment]
@Yuki2718 Sorry, but can you reproduce?

**Comment by Yuki2718** [comment]
Blocking `d38psrni17bvxu.cloudfront.net` makes the page empty which originally showed cookie consent. I worry potential false positive.

**Comment by Daxy7** [comment]
`infosayz.com` Is used for malware redirection based on a quick Google search `google.com/search?q=infosayz.com` So can we add this to uBO's badware filterlist?

**Comment by Yuki2718** [comment]
Maybe? But I can't reproduce.

**Comment by Yuki2718** [comment]
I see only 502 Bad Gateway with or without VPN.

**Comment by Daxy7** [comment]
Can we add `||ifdbdp.com^`? `iyfbodn.com` shares the same HTML document 
```
https://ifdbdp.com/
https://iyfbodn.com/
```
Found alt link for `cdn-image`  at `http://www.carmen-isaila.com/` via [VirusTotal.](https://www.virustotal.com/gui/ip-address/208.91.197.132/relations) likely not needed
<details>

```
<!--
	top.location="https://ifdbdp.com/?fp=";
	/*
-->
<html data-adblockkey=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
				  <meta name="viewport" content="width=device-width"><meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7"></head><frameset rows="100%,*" frameborder="no" border="0" framespacing="0">
	<frame src="https://ifdbdp.com/?fp=">
</frameset>
<noframes>
	<body bgcolor="#ffffff" text="#000000">
	<a href="https://ifdbdp.com/?fp=">Click here to proceed</a>.
	</body>
</noframes></html><!--
*/
-->
```

```
<!--
	top.location="https://iyfbodn.com/?fp=";
	/*
-->
<html data-adblockkey=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
				  <meta name="viewport" content="width=device-width"><meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7"></head><frameset rows="100%,*" frameborder="no" border="0" framespacing="0">
	<frame src="https://iyfbodn.com/?fp=">
</frameset>
<noframes>
	<body bgcolor="#ffffff" text="#000000">
	<a href="https://iyfbodn.com/?fp=">Click here to proceed</a>.
	</body>
</noframes></html><!--
*/
-->
```

</details>

**Comment by Yuki2718** [comment]
Are they needed? On my end 100% blocked by `/px.js?ch=$script`.
